app/boards/shields/clog/clog_left.conf

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) 2020 The ZMK Contributors
```

Comment:
Please remove this `.conf` as well as the `_right.conf`. Only the base `.conf` should be used.